### PR TITLE
Makes chatbox display the latest messages

### PR DESCRIPTION
**Pull request by mike-lvov**
closes https://github.com/sparkletown/internal-sparkle-issues/issues/176

https://user-images.githubusercontent.com/63194656/158488722-c0434e3e-0070-481b-87a5-a27abee5f8aa.mov

### CODE REVIEW

File: src/components/molecules/Chatbox/Chatbox.module.scss [hunk 1]
```diff
@@ -7,6 +7,8 @@
 }
 
 .chatboxMessages {
+  display: flex;
+  flex-direction: column-reverse;
```
Comment (goran-peoski-work) [inline src/components/molecules/Chatbox/Chatbox.module.scss:11]: I don't think depending on CSS is the way to go for displaying data properly. As a rule of thumb, data should be in order even if no styles are applied. 

So, can it be done in a way like `itemsToDisplay.sort().reverse()` or similar before outputing?

I mean, there are even nicer utilities from Flexbox, like `order` CSS property, but if we can display the data properly without resorting to style, I think we should use that.

**Review by goran-peoski-work — CHANGES_REQUESTED** [review]
I don't think just CSS reverting of the sort is the way to go. There might be a simple JS way to do it as well `itemsToDisplay.reverse()`

**Review by goran-peoski-work — APPROVED** [review]
I took half an hour trying to figure this out. There might be something to do with the `inverse` prop of `<InfiniteScroll>` but this PR does solve the issue, even if in the CSS layer.

**Comment by whoaitsaimz** [comment]
I dont think #176 is the correct issue number, but it didn't seem to auto-close it anyway, so prob not a big deal 

...unless someone else at Sparkle is as curious as me and clicks into related issues ;-)

**Pull request merged**